src/components/App/App.js

```diff
@@ -1,26 +1,34 @@
 import './App.scss';
-import { useEffect, useState } from 'react';
-import { getPictureOfDay, getIncomingNearEarthObjects } from '../../apiCalls';
+import { useState } from 'react';
 import IncomingObjects from '../IncomingObjects/IncomingObjects';
 import Nav from '../Nav/Nav';
+import EventDetails from '../EventDetails/EventDetails';
 import ImageOfDay from '../ImageOfDay/ImageOfDay';
 import { Routes, Route } from 'react-router-dom'
 
 function App() { 
-  const [networkError, setNetworkError] = useState(false)
+  const [networkError, setNetworkError] = useState(false);
```

Comment:
While it's working fine, I follow an approach where I set all errors to true and only if things work right, I set them to true. (upto you to pick. Functionally I'm sure it works the same)

Reply:
Thanks for the tip! I like setting it to false so that I can reset it with the useeffect return statement when the component unmounts.